packages/server/graphql/public/typeDefs/gitlab.graphql

```diff
@@ -1 +1,7 @@
 type _xGitLabIssue implements TaskIntegration
+
+type _xGitLabProject implements RepoIntegration
+
+extend type _xGitLabProject {
+  service: IntegrationProviderServiceEnum!
```

Comment:
-1 The resolver for this field is missing. You can test it by adding service to your request in `NewGitLabIssueInput`. Check https://github.com/ParabolInc/parabol/blob/b7e062ce11f6886042779753ee06d01cbd84c932/packages/server/graphql/public/types/_xGitHubIssue.ts for an example